packages/fxa-graphql-api/src/config.ts

```diff
@@ -267,6 +267,68 @@ const conf = convict({
       },
     },
   },
+  recoveryPhone: {
+    enabled: {
+      default: false,
+      doc: 'Enable recovery phone feature',
+      env: 'RECOVERY_PHONE__ENABLED',
```

Comment:
There are multiple `__` (two underscore characters) separators in this file.  Is that a new convention?

Reply:
Yeah, that is equivalent to a `.`.